indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -333,6 +345,7 @@ def _process(self, args):
 
         check_repeat_frequency(self.booking.repeat_frequency, new_booking_data['repeat_frequency'])
 
+        new_booking: typing.Optional['Reservation'] = None
```

Comment:
Would this work?

```suggestion
        new_booking: typing.Optional[Reservation] = None
```

Reply:
Sure, updated in https://github.com/indico/indico/pull/6126/commits/07b7a5b4c8cfa60b78a1729483d63164aae66801, thanks!